### PR TITLE
updated the extension b2b installation command and added info

### LINKED ISSUE #8114: B2B installation requires specific version if Magento is not latest
# General issue

Topic: https://devdocs.magento.com/extensions/b2b/

## Description:

When I install B2B onto Magento 2.3.6 with `composer require magento/extension-b2b`, it loads 1.3 version (`Using version ^1.3 for magento/extension-b2b`), which is the version compatible with Magento 2.4. Though it works when I specify a B2B version such as `composer require magento/extension-b2b:1.1.6`.

## Possible solutions:

Update the text and command or add a note that compatible version must be provided in the command, and add a reference link to the topic with a list of compatible versions _Magento - B2B_.

### PR DESCRIPTION
**Pull request by chiranjeevi-cj**
## Purpose of this pull request

This pull request (PR) added a note that a compatible version must be provided in the command.

## Affected DevDocs pages

-  https://devdocs.magento.com/extensions/b2b/

## Links to Magento source code

-  Fixes #8114 
Screenshot from 2020-12-03 14-19-12

### CODE REVIEW

File: src/extensions/b2b/index.md [hunk 1]
```diff
@@ -14,8 +14,11 @@ For {{site.data.var.ece}} projects, see [Set up Magento B2B module]({{ site.base
 
 1. Change to your Magento installation directory and enter the following command to update your `composer.json` file and install the {{site.data.var.b2b}} extension:
 
+{:.bs-callout-info}
+Compatible Magento - B2B version must be provided in the command.
```
Comment (BarnyShergold) [inline src/extensions/b2b/index.md:18]:
```suggestion
Compatible Magento B2B version must be provided in the command.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Nice update - just a small correction

File: src/extensions/b2b/index.md (hunk 1 above)
Comment (chiranjeevi-cj) [reply src/extensions/b2b/index.md]: @BarnyShergold Updated the changes. Thanks.

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Comment by chiranjeevi-cj** [comment]
@meker12 need to add this https://devdocs.magento.com/release/availability.html#compatibility link?

**Comment by meker12** [comment]
running tests

**Pull request merged**